src/components/Categories.jsx

```diff
@@ -1,7 +1,22 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
 
-const Categories = () => (
-  <div className="categories" />
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+  const status = useSelector((state) => state.status);
+  return (
+    <div className="categories">
+      <p>{status}</p>
+      <input
```

Comment:
- [ ] Again, this is working fine, but it is a project requirement to do this through a button. So please update this